README.md

```diff
@@ -28,7 +28,7 @@ Refer to the table below for a detailed view of all models materialized by defau
 | [fivetran_log__connector_status](https://fivetran.github.io/dbt_fivetran_log/#!/model/model.fivetran_log.fivetran_log__connector_status)        | Each record represents a connector loading data into a destination, enriched with data about the connector's data sync status.                                          |
 | [fivetran_log__transformation_status](https://github.com/fivetran/dbt_fivetran_log/blob/main/models/fivetran_log__transformation_status.sql)     | Each record represents a transformation, enriched with data about the transformation's last sync and any tables whose new data triggers the transformation to run. |
 | [fivetran_log__mar_table_history](https://fivetran.github.io/dbt_fivetran_log/#!/model/model.fivetran_log.fivetran_log__mar_table_history)     | Each record represents a table's active volume for a month, complete with data about its connector and destination.                             |
-| [fivetran_log__credit_mar_destination_history](https://fivetran.github.io/dbt_fivetran_log/#!/model/model.fivetran_log.fivetran_log__credit_mar_destination_history)    | Each record represents a destination's consumption by showing its MAR, total credits used, and credits per millions MAR.                             |
+| [fivetran_log__credit_mar_destination_history](https://fivetran.github.io/dbt_fivetran_log/#!/model/model.fivetran_log.fivetran_log__usage_mar_destination_history)    | Each record represents a destination's consumption by showing its MAR, total credits used, and credits per millions MAR.                             |
```

Comment:
Thanks so much for opening this PR @ferrap! You are correct that this should be updated, thanks for applying the update to the link 🙌 

Additionally, we should probably update the name of the model and the description to be consistent with the information [here](https://github.com/fivetran/dbt_fivetran_log/blob/9f89183075bf03938f81044b673cfb231e353443/models/fivetran_log.yml#L113-L117). Would you be open to applying those updates? Once you do, I can approve and merge!